features/log_in.feature

```diff
@@ -0,0 +1,18 @@
+Feature: As a visitor.
+So that I can use the service.
+I would like to be able to go to login page and login on my account.
+
+Background:
+  Given the following users exist
+    | name  | email           | password     |
+    | felix | felix@telia.com | fisk1234     |
+
+Scenario: As I login
+  Given I am on the index page
+  And I click "Login" button
```

Comment:
Wait, why do you have two steps: `I click "Login" button` and `I click "Log in"`. These do the same thing, right?